Models/Sequence.cs

```diff
@@ -34,6 +34,7 @@ public Sequence(BamAlignment bamAlignment)
             Identifier = new ReadOnlyMemory<byte>(bamAlignment.read_name).ToArray();
             Read = new ReadOnlyMemory<byte>(bamAlignment.seq).ToArray();
             Quality = new ReadOnlyMemory<byte>(bamAlignment.qual).ToArray();
+            Blank = Array.Empty<byte>();
```

Comment:
this is likely better anyway, the "blank" doesn't always mean blank, it's sometimes a duplicate of the identity line